www/unstable/userguide/weekyears.md

```diff
@@ -0,0 +1,291 @@
+"2016, February, 4" is equivalently week-year 2016, week 5,
+Thursday.
+
+The two representations are exactly equivalent: every day
+can be represented in either form. Week-years are more widely used
+in some countries than in others, typically in business or
+educational settings.
+
+Noda Time features for week-years
+===
+
+As of Noda Time 2.0, week-year rules are entirely separate from calendar
+systems. A rule is represented by the
+[`IWeekYearRule`](noda-type://NodaTime.Calendars.IWeekYearRule)
+interface, and built-in implementations are obtained via factory
+members in the
+[`WeekYearRules`](noda-type://NodaTime.Calendars.WeekYearRules)
+static class.
+
+Although the `GetWeeksInWeekYear` and `GetLocalDate` methods accept
+`CalendarSystem` parameters, extension methods in
+[`WeekYearRuleExtensions`](noda-type://NodaTime.Calendars.WeekYearRuleExtensions)
+are equivalent but without the parameters, simply delegating to the
+parameterized forms and passing in the ISO calendar system. This
+reduces the burden of implementing `IWeekYearRule`, but allows
+callers who are only using the ISO calendar system to omit it.
+
+There are two methods to convert from year-month-day representation
+to week-based representation: `GetWeekYear` and `GetWeekOfWeekYear`,
+both accepting a `LocalDate`. There is no equivalent to obtain the
+day of week, as that does not change based on the rule or even on
+the calendar system.
+
+Noda Time 2.0 comes with two kinds of rule: ISO-like rules, and BCL
+rules.
+
+ISO-like rules
+---
+
+Let's start with the most common rule: `WeekYearRules.Iso`, which
+implements the rule specified in ISO-8601. Under this rule:
+
+- All weeks are Monday-to-Sunday
+- If the week that contains the first day of the year has four days
+  or more in that year, then the whole of that week is in week 1 of
+  that year; otherwise, the whole of that week is in the final week
+  of the previous year.
+  
+To put the "four days or more" rule in a different way: if the first
+day of the calendar year is Monday, Tuesday, Wednesday or Thursday,
+then the week containing that day is week 1 of the same year; if
+the calendar year starts on a Friday, Saturday or Sunday, then that
+week is in the previous week-year.
+
+Examples:
+
+- January 1st 2013 is a Tuesday, so 2013 week 1 runs from 2012-12-31
+to 2013-01-06 inclusive
+- January 1st 2014 is a Wednesday, so 2014 week 1 runs from 2013-12-30
+to 2013-01-05 inclusive
+- January 1st 2015 is a Thursday, so 2015 week 1 runs from
+2014-12-29 to 2015-01-04 inclusive
+- January 1st 2016 is a Friday, so **2015 week 53** runs from
+2015-12-28 to 2016-01-03 inclusive, and 2016 week 1 runs from
+2016-01-04 to 2016-01-10 inclusive.
+- January 1st 2017 is a Sunday, so **2016 week 52** runs from
+2016-12-26 to 2017-01-01 inclusive, and 2017 week 1 runs from
+2017-01-02 to 2017-01-08 inclusive.
+
+Once you understand this, it's not terribly hard to add two other
+axes of flexibility:
```

Comment:
I'd like a clearer bridge sentence between "this is _the_ ISO calendar system" and "here are some ISO-like calendars we can construct".

Perhaps replace this with something like "ISO-8601 only defines a single rule for calculating week years, but it is possible to vary some of the parameters to produce ISO-like rules:"